docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+services:
+  backend:
+    build:
+      context: ./ninjareact
+    container_name: django_backend
+    command: python manage.py runserver 0.0.0.0:8000
+    ports:
+      - "8000:8000"
+    volumes:
+      - ./ninjareact:/app
```

Comment:
application code is included in the Docker image  by your Dockerfile and should not be mapped into the container at runtime.